### PR TITLE
WE-310 Decrease build times

### PR DESCRIPTION
**Pull request by jonambas**
> Build time: 7m 9s. Total deploy time: 11m 59s

**What Changed**
- Removes nav data sourcing from `getStaticProps`
- Adds `yaml-loader` and loads nav data via webpack instead of on each route

**How to verify**
- Run the site locally or visit the deploy preview
- Verify the navigation works correctly
- Verify build times are lowered in Netlify

### CODE REVIEW

File: components/site/momentumNavigation.tsx [hunk 1]
```diff
@@ -15,101 +23,126 @@ export interface MomentumNavigationItemProps {
...
-  }
-
 
   return (
     <Box width="260px" position="sticky" top="0">
       <StyledLink $active={environment?.location?.pathname === '/momentum'}>
         <Link href="/momentum" passHref>
-          <Box as="a" display="inline-block" fontSize="200" fontWeight="semibold" py="200" lineHeight="200" px="500">Momentum Documentation</Box>
+          <Box
+            as="a"
```
Comment (jonambas) [inline components/site/momentumNavigation.tsx:68]: Idk why there are prettier changes here, do we need to adjust our setup?

File: package.json [hunk 2]
```diff
@@ -84,6 +84,7 @@
     "prettier": "2.4.1",
     "start-server-and-test": "^1.14.0",
     "strip-markdown": "^5.0.0",
-    "typescript": "4.4.4"
+    "typescript": "4.4.4",
+    "yaml-loader": "^0.6.0"
```
Comment (jonambas) [inline package.json:88]: Need to uninstall js-yaml

File: components/site/momentumNavigation.tsx [hunk 3]
```diff
@@ -2,11 +2,19 @@ import React from 'react';
...
+  PeopleOutline,
+  Code,
+  DeviceHub,
+} from '@sparkpost/matchbox-icons';
 import styled from 'styled-components';
 import css from '@styled-system/css';
-import { tokens} from '@sparkpost/design-tokens';
+import { tokens } from '@sparkpost/design-tokens';
 import useStatus from 'hooks/useStatus';
+import data from 'content/momentum/navigation.yml';
```
Comment (jonambas) [inline components/site/momentumNavigation.tsx:17]: This is essentially all i added to this file

File: components/site/momentumNavigation.tsx (hunk 1 above)
Comment (logansparlin) [reply components/site/momentumNavigation.tsx:68]: Not sure why either, maybe I need to check my local setup

**Review by logansparlin — APPROVED** [review]
Looks good!

**Pull request merged**